backend/inspirehep/records/api/mixins.py

```diff
@@ -175,7 +197,7 @@ def add_local_file(self, key, bucket=None):
         file_object = self.get_file_object(key, bucket)
```

Comment:
When file_object will be None don't you want to re-download file form original_url if it's possible?

Reply:
You should also add test for re-adding deleted file (we talked about it)